src/Platform/Scope/HttpScope.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
```

Comment:
I like it! Let’s name this class only HTTP. ‘Scope’ is already in the namespace.